centaur/src/main/resources/standardTestCases/wdl_biscayne/biscayne_new_engine_functions/biscayne_new_engine_functions.wdl

```diff
@@ -15,8 +15,15 @@ workflow biscayne_as_map_et_al {
 
   Array[Pair[String,Int]] z_pairs_in = [("a", 1), ("b", 2), ("a", 3)]
 
+  Int smallestInt = 1
+  Float smallFloat = 2.718
+  Float bigFloat = 3.141
+  Int biggestInt = 10
```

Comment:
Does it make sense to test some extremes like int max and float max? Which is larger, int max or float max? I don't know the answer, but we should do the right thing.

Reply:
I added the test. It turns out that max float is significantly larger than max int, and the max-ing of int X float gives you float, so I think we're safe.